apps/core/settingmanager.cpp

```diff
@@ -31,6 +31,7 @@ SettingManager::SettingManager(QObject *parent)
     m_clearCookiesConfItem = new MGConfItem("/apps/sailfish-browser/actions/clear_cookies", this);
     m_clearPasswordsConfItem = new MGConfItem("/apps/sailfish-browser/actions/clear_passwords", this);
     m_clearCacheConfItem = new MGConfItem("/apps/sailfish-browser/actions/clear_cache", this);
+    m_clearSitePermissionsConfItem = new MGConfItem("/apps/sailfish-browser/actions/clear_site_permissions", this);
```

Comment:
It's conceptually wrong that we keep on adding these confitems. These were here originally here because we had browser settings separated to the settings app. Hence, we needed a persistent mechanism to handle these kind of changes also when browser was not running. Now, however, all settings are part of the browser and accessible only when browser is running implying the there is no need to route setting change via confitem.

Now it would be probably good time create a PR that cleans up SettingsManager and exposes it to QML as singleton with needed invokables so that PrivacySettingsPage.qml could be changed like this (nb BookmarkManager should be exposed as well):

```
                        remorse = Remorse.popupAction(page, qsTrId("settings_browser-la-cleared_private_data"),
                                                 function() {
                                                     if (clearHistory.checked) {
                                                         SettingsManager.clearHistory()
                                                     }
                                                     if (clearCookies.checked) {
                                                         SettingsManager.clearCookies()
                                                     }
                                                     if (clearSavedPasswords.checked) {
                                                         SettingsManager.clearPasswords()
                                                     }
                                                     if (clearCache.checked) {
                                                         SettingsManager.clearCache()
                                                     }
                                                     if (clearBookmarks.checked) {
                                                         BookmarkManager.clearBookmarks()
                                                     }
                                                     if (clearSitePermissions.checked) {
                                                         SettingsManager.clearSitePermissions()
                                                     }
                                                 }
                        );
```